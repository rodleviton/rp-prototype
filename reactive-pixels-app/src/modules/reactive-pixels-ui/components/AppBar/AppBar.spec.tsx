import * as React from 'react';
import { render } from 'react-testing-library';
import WithTheme from '../../utils/test/WithTheme';
import { AppBar } from './';

describe('<AppBar />', () => {
  it('should render <AppBar /> component', () => {
    const { container } = render(
      <WithTheme>
        <AppBar />
      </WithTheme>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
