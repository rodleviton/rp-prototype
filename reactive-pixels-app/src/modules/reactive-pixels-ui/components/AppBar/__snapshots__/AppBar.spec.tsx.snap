// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppBar /> should render <AppBar /> component 1`] = `
<div
  class="css-g713d"
/>
`;
